dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/test/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesExportTest.java

```diff
@@ -167,11 +201,13 @@ public void shouldAddServiceSpec(RuntimeType rt) throws Exception {
                 deployment.getSpec().getTemplate().getSpec().getContainers().get(0).getPorts().get(0).getContainerPort());
 
         Service service = getService(rt);
+        List<ServicePort> ports = service.getSpec().getPorts();
         Assertions.assertEquals("route-service", service.getMetadata().getName());
-        Assertions.assertEquals(1, service.getSpec().getPorts().size());
-        Assertions.assertEquals("http", service.getSpec().getPorts().get(0).getName());
-        Assertions.assertEquals(80, service.getSpec().getPorts().get(0).getPort());
-        Assertions.assertEquals("http", service.getSpec().getPorts().get(0).getTargetPort().getStrVal());
+        Assertions.assertEquals("NodePort", service.getSpec().getType());
+        Assertions.assertEquals(1, ports.size());
+        Assertions.assertEquals("http", ports.get(0).getName());
+        Assertions.assertEquals(8080, ports.get(0).getPort());
```

Comment:
the Kubernetes service should be exposed on port 80 by default so other components in the cluster can call the service with its pure domain name URL (e.g. `http://my-service.default/` instead of `http://my-service.default:8080/`